tsconfig.json

```diff
@@ -113,6 +113,7 @@
 	  "node_modules",
 	  ".vscode-test",
 	  "out/**/*",
+	  "test_project/**/*",
```

Comment:
@orukRed
これを消せば私の環境で動きました

``` 
error TS5055: Cannot write file '**/tyranosyntax/test_project/data/system/KeyConfig.js' because it would overwrite input file.
```